src/driver/src/DefaultDriver.cpp

```diff
@@ -87,6 +88,63 @@ void reportMetrics(genny::metrics::Registry& metrics,
     actorSetup.report(std::move(finishTime), std::move(duration), std::move(outcome));
 }
 
+/**
+ * Filter to remove any nodes inside dedicated "ignore" values.
+ */
+UnusedNodes removeIgnored(const UnusedNodes& unused, const std::vector<std::string>& ignored) {
```

Comment:
Even though I wrote this I still can't easily visually+mentally parse it. I shudder to think what this would have looked like in C++11 though 😂

I think my original impl had the python equivalent explaining the logic. Subjective so fine to ignore (and I'm just lurking on this PR anyway) but consider explaining what this is doing just a bit.